### PR TITLE
[Jormungandr] Set parameter for region

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVITIAII-3013

When no coverage is set in a request, the parameters from the chosen coverage needs to be applied.

Also:
Some changes in comments

### CODE REVIEW

**Review by kadhikari — APPROVED** [review]
It seems that we didn't have a test on default parameter without coverage. There is not any test failure after this modification.

**Comment by mbounabi** [comment]
> It seems that we didn't have a test on default parameter without coverage. There is not any test failure after this modification.

@kadhikari Indeed. However, there's no need to apply default values in any case as, in the end, there will always be a coverage to be queried on.

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -499,7 +493,7 @@ def _set_specific_params(mod):
         responses = {}
         for r in possible_regions:
             self.region = r
-
+            _set_specific_params(i_manager.instances[r])
```
Comment (nonifier) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:496]: :+1:

**Review by nonifier — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/journey_common.py [hunk 2]
```diff
@@ -127,16 +127,15 @@ def compute_regions(args):
...
+    :return:    If region is set in the query, return it
+                If not, return a list of possible regions for the query
+    """
     if region:
-        region_obj = i_manager.get_region(region)
-
-    if not region:
-        # TODO how to handle lon/lat ? don't we have to override args['origin'] ?
-        possible_regions = compute_regions(args)
+        return [i_manager.get_region(region)]
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/journey_common.py:135]: :+1:

**Review by xlqian — APPROVED** [review]

**Pull request merged**